cmd/exec_test.go

```diff
@@ -267,7 +267,9 @@ main contains "hello" if {
 					defer cancel()
 					go func() {
 						err := runExecWithContext(ctx, params)
-						if err != nil {
+						// Note(philipc): Catch the expected cancellation
+						// errors, allowing unexpected test failures through.
+						if err != context.Canceled {
```

Comment:
Here's where we adjust our test suite to deal with context-cancellation errors bubbling up.